assets/constants.ts

```diff
@@ -7057,6 +7073,6 @@ export default {
   ],
   "CONTRIBUTOR_CERTIFICATE_WIDTH": 1493,
   "CONTRIBUTOR_CERTIFICATE_HEIGHT": 1313,
-  "BRANCH_NAME": "",
-  "SHORT_COMMIT_HASH": ""
+  "BRANCH_NAME": "shifted-feedback-updates-with-feature-flag",
```

Comment:
I don't think you meant to commit these changes. (Please check your "Files Changed" tab carefully and make sure all the changes are correct before re-requesting a review.)

Reply:
Done 
Thanks!